package.json

```diff
@@ -9,6 +9,7 @@
   "dependencies": {
     "bluebird": "3.5.0",
     "le-acme-core": "https://git.daplie.com/rolodato/le-acme-core/repository/archive.tar.gz",
+    "moment": "^2.20.1",
```

Comment:
I don't think it's necessary to introduce this dependency, we can do a little date arithmetic instead :)

Reply:
removed the dependency